### ISSUE TITLE
Adding version information to requirements.txt

### ISSUE THREAD

**Issue by Ryuno-Ki** [pitchmuc/pylaunch#1]
Hi Julien,

since you are writing a library, you should [add version hints on your requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files). You can use `pip freeze` for that.

This way, you can utilise tools to check for breaking changes caused by your dependencies.
Plus, it makes debugging easier, since you can more or less be sure to run the same version than the person opening an issue.

**Comment by pitchmuc** [comment]
Thanks for the tip. 
I'll do that soon then. I let that thread open so I can close it when I documented the minimum version of my dependencies. 
Thank you very much for all the great tips. 

I hope you have seen that I have updated the library since our presentation and there are cool features now, like "extractAnalyticsConfig"

**Comment by pitchmuc** [comment]
I am starting to write the wrapper for the Analytics API 2.0 now. 
I hope to be finished by the end of the year (mostly time constraint than difficulties). 
After that, I'll try to create a global adobe marketing cloud that you can download with pypi and that can be ran directly. 
Stay tuned. ;)

**Comment by VikashKotharyWork** [comment]
I've created a PR #10 for adding a poetry `pyproject.toml` to the repo. I find it's a lot more powerful compared to the traditional `requirements.txt` as you have lock files and can separate dev and library requirements. What do you think @pitchmuc @Ryuno-Ki ?
See https://python-poetry.org/ for more information.

**Comment by Ryuno-Ki** [comment]
More powerful also implies a steeper learning curve.
However, I never used Poetry or Make myself.

**Comment by VikashKotharyWork** [comment]
TLDR; `poetry add pylaunch`, `poetry add --dev pylaunch` and `poetry install`. 
No need for version information thanks to `poetry.lock`.

**Comment by pitchmuc** [comment]
I did add the version requirements for the publishing to pypi.
